### DISCUSSION TITLE
API to fetch Oauth Apps

### DISCUSSION THREAD

**Discussion by mudit-pathak-sp** [community/community#33674] [Apps, API and Webhooks]
Does Github has any API to fetch Oauth apps/Third Party Applications?

**Comment by byrneh** [comment]
@mudit-pathak-sp, unfortunately not, there is one for retreiving Github Apps.  
You are not the first person and will not be the last one to ask or want this,

**Reply by mudit-pathak-sp** [reply]
Yeah saw that in the documentation. Thanks @byrneh !
